### PR TITLE
Added ignoreBlacklistedUsers and ignoreBlacklistedGuilds Options

### PR DESCRIPTION
**Pull request by AdityaTD**
### Description of the PR
Adds `ignoreBlacklistedUsers` & `ignoreBlacklistedGuilds` to Monitors

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)
- Added ignoreBlacklistedUsers to monitors
- Added ignoreBlacklistedGuilds to monitors

### Semver Classification

- [ ] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) framework interface.
- [x] This PR adds methods or properties to the framework interface.
- [ ] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

**Comment by bdistin** [comment]
need to add defaults to constants, and remove the blacklisted inhibitor still

File: src/lib/structures/Monitor.js [hunk 1]
```diff
@@ -88,7 +104,9 @@ class Monitor extends Piece {
 			!(this.ignoreSelf && this.client.user === message.author) &&
 			!(this.ignoreOthers && this.client.user !== message.author) &&
 			!(this.ignoreWebhooks && message.webhookID) &&
-			!(this.ignoreEdits && message._edits.length);
+			!(this.ignoreEdits && message._edits.length) &&
+			!(this.ignoreBlacklistedUsers && this.client.configs.blacklistedUser.includes(message.author.id)) &&
```
Comment (MrJacz) [inline src/lib/structures/Monitor.js]: its `userBlacklist`

File: src/lib/structures/Monitor.js [hunk 2]
```diff
@@ -88,7 +104,9 @@ class Monitor extends Piece {
...
+			!(this.ignoreBlacklistedGuilds && this.client.configs.blacklistedGuilds.includes(message.guild.id));
```
Comment (MrJacz) [inline src/lib/structures/Monitor.js]: its `guildBlacklist`

Comment (AdityaTD) [reply src/lib/structures/Monitor.js]: Thanks for pointing out

**Pull request merged**